anime-relations.txt

```diff
@@ -728,6 +728,9 @@
 - 16498|7442|16498:38-49 -> 35760|13569|99147:1-12!
 # Shingeki no Kyojin -> ~ Season 3 Part 2
 - 16498|7442|16498:50-59 -> 38524|41982|104578:1-10!
+# Shingeki no Kyojin Season 3 -> ~ Part 2
+- 5760|13569|99147:13-22 -> 38524|41982|104578:1-10
+- 5760|13569|99147:50-59 -> 38524|41982|104578:1-10
```

Comment:
`5760` should be `35760` I think in both lines.

Reply:
Thanks for the heads up.